Issue with example code

In the main readme, there's this sample code:

```rs
use fastwebsockets::upgrade::upgrade;
use hyper::{Request, body::{Incoming, Bytes}, Response};
use http_body_util::Empty;
use anyhow::Result;

async fn server_upgrade(
  mut req: Request<Incoming>,
) -> Result<Response<Empty<Bytes>>> {
  let (response, fut) = upgrade::upgrade(&mut req)?;

  tokio::spawn(async move {
    if let Err(e) = handle_client(fut).await {
      eprintln!("Error in websocket connection: {}", e);
    }
  });

  Ok(response)
}
```

For me I get this error
```
error[E0433]: cannot find module or crate `upgrade` in this scope
  --> C:\Users\aaron\Projects\Coding\automata_wars_server\src\lib.rs:42:24
   |
42 |     let (response, fut) = upgrade::upgrade(&mut req)?;
   |                           ^^^^^^^ function `upgrade` is not a crate or module


```

It looks like `use fastwebsockets::upgrade::upgrade;` should be `use fastwebsockets::upgrade;` or `let (response, fut) = upgrade::upgrade(&mut req)?;` should just be `let (response, fut) = upgrade(&mut req)?;`

--- a/README.md
+++ b/README.md
@@ -67,7 +67,7 @@ handshakes.
 This feature is powered by [hyper](https://docs.rs/hyper).
 
 ```rust
-use fastwebsockets::upgrade::upgrade;
+use fastwebsockets::upgrade;
 use hyper::{Request, body::{Incoming, Bytes}, Response};
 use http_body_util::Empty;
 use anyhow::Result;
@@ -109,7 +109,7 @@ async fn connect() -> Result<WebSocket<Upgraded>> {
     .header(CONNECTION, "upgrade")
     .header(
       "Sec-WebSocket-Key",
-      fastwebsockets::handshake::generate_key(),
+      handshake::generate_key(),
     )
     .header("Sec-WebSocket-Version", "13")
     .body(Empty::<Bytes>::new())?;

